Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -109,14 +110,40 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
       .map { _ in AppEnvironment.current.currentUser }
       .map(isNotNil)
 
+    let notChangingPaymentMethod = context.map { context in
+      context.isUpdating && context != .changePaymentMethod
```

Comment:
Can you change the payment method on a post-campaign pledge? I thought they were immutable.

Reply:
ah yes thank you!